pulp_ansible/app/urls.py

```diff
@@ -72,7 +72,7 @@
 ]
 
 urlpatterns = [
-    path("ansible/collections/", CollectionUploadViewSet.as_view({"post": "create"})),
+    path("ansible/collections/", CollectionVersionViewSet.as_view({"post": "create"})),
```

Comment:
@bmbouter I'm not removing the url, just changing the viewset

Reply:
Ah true. Thank you.